src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.ts

```diff
@@ -186,41 +185,45 @@ export class HearingActualsViewEditPartiesComponent implements OnInit, OnDestroy
   }
 
   private addActualParticipantsAndParties(actualParty: ActualDayPartyModel) {
-    if (this.isPlannedParty(actualParty)) {
-      this.participants.push({
-        name: `${actualParty.individualDetails.firstName} ${actualParty.individualDetails.lastName}`,
-        id: actualParty.actualPartyId
-      });
+    if (actualParty?.individualDetails) {
+      if (this.isPlannedParty(actualParty)) {
+        this.participants.push({
+          name: `${actualParty.individualDetails.firstName} ${actualParty.individualDetails.lastName}`,
+          id: actualParty.actualPartyId
+        });
+      }
+      this.parties.push(this.fb.group({
+        firstName: [actualParty.individualDetails.firstName, [this.validators.mandatory('Enter first name')]],
+        lastName: [actualParty.individualDetails.lastName, this.isPlannedParty(actualParty) ? [] : [this.validators.mandatory('Enter last name')]],
```

Comment:
@johnbenjamin-hmcts - is the text `Enter last name` going to get translated to Welsh somehow, or is that not required? Same question for the other validators below

Reply:
@Daniel-Lam - I have not changed this line of code as part of this fix, just wrapped it around with an _if_ statement. Might be there is another ticket to handle the Welsh for Hearings, but I am not sure.
I will check with our BA regarding this.